### DISCUSSION TITLE
:shallow_pan_of_food:  Restaurant menu teller fully digitalized 😋

### DISCUSSION THREAD

**Discussion by adriens** [suno-ai/bark#333] [Show and tell]
:wave: hello from Nouméa:

# :memo: Full article

https://dev.to/adriens/agi-bark-smart-waitress-285h
![image](https://github.com/suno-ai/bark/assets/5235127/c44a461e-30f3-4d5e-be8e-32b07e637ad5)

# :popcorn: Video

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/t6tDH8qaVSA/0.jpg)](https://www.youtube.com/watch?v=t6tDH8qaVSA)

**Comment by wenchochooo** [comment]
hahaha
